src/bench.c

```diff
@@ -97,6 +122,25 @@ int main(int argc, char** argv) {
     int d = argc == 1;
     int iters = get_iters(20000);
 
+    /* Check for invalid user arguments */
+    char* valid_args[] = {"ecdsa", "verify", "ecdsa_verify", "sign", "ecdsa_sign", "ecdh", "recover",
+                         "ecdsa_recover", "schnorrsig", "schnorrsig_verify", "schnorrsig_sign"};
+    int valid_args_size = 11;
+    int invalid_args = have_invalid_args(argc, argv, valid_args, valid_args_size);
+
+    if(argc > 1) {
+        if(have_flag(argc, argv, "-h")
+           || have_flag(argc, argv, "--help")
+           || have_flag(argc, argv, "help")) {
+            help(argv);
+            return 1;
```

Comment:
`ls --help` does say:

> Exit status:
 0  if OK,
 1  if minor problems (e.g., cannot access subdirectory),
 2  if serious trouble (e.g., cannot access command-line argument).

I used `return 1` here since the [bench_ecmult.c](https://github.com/bitcoin-core/secp256k1/blob/master/src/bench_ecmult.c#L295) followed this conversion.

Shall I also modify `bench_ecmult's` help to `return 0`?

Reply:
Oh indeed, I think that's wrong in bench_ecmult too

Reply:
Okay, will update both.